cmd/clusterctl/client/repository/components.go

```diff
@@ -292,9 +301,73 @@ func addNamespaceIfMissing(objs []unstructured.Unstructured, targetNamespace str
 	return objs
 }
 
+func fixWebHookTargetNamespace(o unstructured.Unstructured, targetNamespace string) (unstructured.Unstructured, error) {
+	if o.GetKind() == mutatingWebhookConfigurationKind || o.GetKind() == validatingWebhookConfigurationKind {
```

Comment:
Could it be that we also have this annotation on the CRDs?

Reply:
great catch, yes it looks like it. I'll fix straight away..